parm/soca/berror/soca_ensrecenter.yaml

```diff
@@ -4,104 +4,60 @@ geometry:
   mom6_input_nml: mom_input.nml
   fields metadata: ./fields_metadata.yaml
 
-date: '{{ ATM_WINDOW_BEGIN }}'
+date: '{{ MARINE_WINDOW_END_ISO }}'
 
-layers variable: [hocn]
+layers variable: [sea_water_cell_thickness]
 
 # TODO(AFE) fix ice recentering in cycled da
-#increment variables: [tocn, socn, uocn, vocn, ssh, hocn, cicen, hicen, hsnon]
-increment variables: [tocn, socn, uocn, vocn, ssh, hocn]
-
-set increment variables to zero: [ssh]
+increment variables:
+- sea_water_potential_temperature
+- sea_water_salinity
+- eastward_sea_water_velocity
+- northward_sea_water_velocity
+set increment variables to zero:
+- eastward_sea_water_velocity
+- northward_sea_water_velocity
```

Comment:
Is the change of variables (compared to develop) here intentional?
For my education, what does `set increment variables to zero` mean and when would one use it?

Reply:
we use it when writing out the mom6 iau increment (for now), and to zero out the background error for selected variables.

Reply:
why zero out water velocities in this case?

Reply:
It's never shown to be beneficial so far but we plan to revisit this when running with 80 members. We were also getting increments that were way too large when we were using a "climatological" ensemble.